docker/influxdb/setup-influxdb.sh

```diff
@@ -0,0 +1,5 @@
+#! /bin/sh
+apk add curl \
+  && timeout -t 300 bash -c 'while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' 127.0.0.1:8086/ping)" != "204" ]]; do echo "Testing influxdb connection" && sleep 5; done' || false \
```

Comment:
Nit: curl has a retry facility (few `--retry*` options).

Reply:
Cool. I think either way works but i'm open to changing it.

Reply:
Ended up doing:
```
#! /bin/sh
apk add curl \
  && [[ $(curl -s -o /dev/null -w '%{http_code}' --connect-timeout 5 --max-time 5 --retry 10 --retry-delay 0 http://127.0.0.1:8086/ping) == "204" ]] || false \
  && influx -execute "CREATE DATABASE $INFLUXDB_IML_DB" \
  && influx -execute "CREATE DATABASE $INFLUXDB_STRATAGEM_SCAN_DB"
```